backend/src/data/migrations/20220807134704_add_associations.ts

```diff
@@ -0,0 +1,56 @@
+import { Knex } from 'knex';
```

Comment:
Let's give more clear name for migration - `add_groups_permissions_associations` or something like that